app/i18n/it/conf.php

```diff
@@ -105,7 +105,12 @@
 	),
 	'profile' => array(
 		'_' => 'Gestione profili',
-		'api' => 'Gestione API',
+		'api' => array(
+			'_' => 'Gestione API',
+			'check_link' => ''Verificare lo stato dell'API tramite: <kbd><a href="../api" target="_blank">%s</a></kbd>'',
+			'disabled' => 'L'accesso all'API è disabilitato.',
```

Comment:
```suggestion
			'disabled' => 'L\'accesso all\'API è disabilitato.',
```

Reply:
```suggestion
			'disabled' => 'L’accesso all’API è disabilitato.',
```
I saw that you want to use a different method from mine